src/windows-manager.cpp

```diff
@@ -887,6 +887,10 @@ namespace viewer {
                 itg->second->removeChild(n);
             }
             nodes_.erase (nodeName);
+	    // if node is of type collada, explicitely remove it from cache
+	    LeafNodeColladaPtr_t lnc(dynamic_pointer_cast<LeafNodeCollada>
+				     (n));
+	    if (lnc) lnc->removeFromCache();
```

Comment:
I don't think we want to do this systematically.

Reply:
In my application, I use a temporary file '/tmp/octree.obj' representing an octree built from a point cloud. I want to be able to update the octree with new measurements. For that, I need to remove the previous one from the cache otherwise, `gepetto-gui` will not reload the new file (with the same name, but with a different content).

If a user wants to remove a collada node from the tree without deleting it, he can use method `removeFromGroup`.

Reply:
I regularly delete a node and recreate it to reset its properties to zero. `removeFromGroup` cannot do this.

For your case, I think it is better to:
- add a VoidProperty to LeafNodeCollada that is bound to `removeFromCache` or
- add a `removeFromCache` method to the IDL.

The first option is the simplest to implement, although the feature will be a bit hidden.

Reply:
I will implement the second one.

Reply:
I have just pushed the required modification.